setup.py

```diff
@@ -18,6 +18,8 @@
     license=package['license'],
     description=package['description'] if 'description' in package else package_name,
     install_requires=[
-        'colour==0.1.5'
+        'dash',
+        'dash_html_components',
+        'dash_renderer',
```

Comment:
I think that this should technically be `dash-html-components` and `dash-renderer`

Reply:
Yes, good catch! Updating this. now

Reply:
I think the reason for this error is that I copied directly from dash's install_requires:
https://github.com/plotly/dash/blob/3533073d1e2de5ad208a278d604822afbb9c57fa/setup.py#L19-L23

Looking at other `setup.py` examples, `dash-html-components` is indeed the right way to do it.